### PR TITLE
Add Zaragoza recommendations

### PR DESCRIPTION
**Pull request by PepMorales**

### CODE REVIEW

File: PepMorales/PepMorales-Zaragoza.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Zaragoza :heart:
+
+### Restaurants
```
Comment (githubteacher) [inline PepMorales/PepMorales-Zaragoza.md:3]: @PepMorales  I love to eat! Do you have a favorite burger place? 🍔

Comment (PepMorales) [inline PepMorales/PepMorales-Zaragoza.md:3]: I have added my best recommendations

File: PepMorales/PepMorales-Zaragoza.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Zaragoza :heart:
```
Comment (ghost) [inline PepMorales/PepMorales-Zaragoza.md:1]: @PepMorales Zaragoza spain?   what is the typical food in there?

File: PepMorales/PepMorales-Zaragoza.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+### Restaurants
+- Casa de Manolo
+- Rincon de la Reina
+- Asadero de Aranda
+
```
Comment (ssheru) [inline PepMorales/PepMorales-Zaragoza.md:7]: Really good stuff

File: PepMorales/PepMorales-Zaragoza.md (hunk 1 above)
Comment (ssheru) [inline PepMorales/PepMorales-Zaragoza.md:3]: Which location is that Mr.Pep

Comment (PepMorales) [inline PepMorales/PepMorales-Zaragoza.md:3]: The first 2 places are in the city centre

File: PepMorales/PepMorales-Zaragoza.md (hunk 2 above)
Comment (PepMorales) [inline PepMorales/PepMorales-Zaragoza.md:1]: Yes
the typical food is a bit tricky... I don't know how to translate those dishes into English! :smile:

Comment (PepMorales) [inline PepMorales/PepMorales-Zaragoza.md:1]: @moyomendoza, are you from Mexico?

Comment (ghost) [inline PepMorales/PepMorales-Zaragoza.md:1]: así es,  México,  no hay que traducir :smile:

**Pull request merged**